test/rules/structured-imports/ancestorCases.ts.lint

```diff
@@ -0,0 +1,11 @@
+import * as relative from "./relativeImport1";
```

Comment:
Based on the current state of the custom rule, side-effect imports get treated as any other import statement. 
[See test case](https://github.com/Shopify/tslint-config-shopify/blob/87d92db548d510f337e52c64965b879f8418f335/test/rules/structured-imports/invalid-order-side-effects.ts.lint)